source/FreeRTOS_DHCPv6.c

```diff
@@ -194,8 +132,6 @@ static void vDHCPv6ProcessEndPoint_HandleReply( NetworkEndPoint_t * pxEndPoint,
 static BaseType_t xDHCPv6ProcessEndPoint_HandleAdvertise( NetworkEndPoint_t * pxEndPoint,
                                                           DHCPMessage_IPv6_t * pxDHCPMessage );
 
-static void vDHCPv6ProcessEndPoint_SendDiscover( NetworkEndPoint_t * pxEndPoint );

```

Comment:
This function is useless because it should be never triggered.

Reply:
Can we check why was it added in the first place.

Reply:
This function was designed to re-send the solicitation message when DHCP message pointer is NULL.
Refer to [code of receive advertise](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/dev/IPv6_integration/source/FreeRTOS_DHCPv6.c#LL719C19-L719C33) and [code of receive reply](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/dev/IPv6_integration/source/FreeRTOS_DHCPv6.c#L399), we can't handle any coming DHCP message without it.
So I add a flow to stop whole DHCPv6 flow when allocate fail.
Then this function `vDHCPv6ProcessEndPoint_SendDiscover` is never called.